src/SupabaseClient.ts

```diff
@@ -59,7 +60,7 @@ export default class SupabaseClient {
     this.schema = settings.schema
 
     this.auth = this._initSupabaseAuthClient(settings)
-    this.realtime = this._initRealtimeClient()
+    this.realtime = this._initRealtimeClient(settings.realtimeOptions)
```

Comment:
@quicksnap let's change `realtimeOptions` to `realtime` to keep it simple:

```suggestion
    this.realtime = this._initRealtimeClient(settings.realtime)
```

Reply:
@quicksnap can you please also add this to function doc: `* @param options.realtime...`

Reply:
All done!